index.html

```diff
@@ -14,239 +14,320 @@
-                            <h4>Skills</h4><img src="./images/chevron-right-solid.svg" alt="works">
-                        </li>
+        <!-- this is the section part -->
 
-                    </ul>
+
+        <section class="first-section">
+
+            <div class="section2">
+                <h1>I'm Ekene, <br> Glad to see you!</h1>
+                <p class="first-paragraph">I'm a software developer! I can help you build a product , feature or website
+                    Look through
+                    some of my work and experience! If you like what you see and have a project you need coded,
+                    don't hesitate to contact me.</p>
+                <p class="second-paragraph">LETS CONNECT</p>
+                <ul class="social-media">
+                    <li> <a href="#"><img src="./images/github.jpg" alt="github"></a> </li>
+                    <li> <a href="#"><img src="./images/medium.jpg" alt="Linkedin"></a> </li>
+                    <li> <a href="#"><img src="./images/twitter.jpg" alt="medium"></a> </li>
+                    <li> <a href="#"><img src="./images/Vector.jpg" alt="twitter"></a> </li>
+                </ul>
+            </div>
+        </section>
+
+        <!-- Hero section -->
+
+        <div class="grid-position">
+            <div class="grid-container">
+                <section class="GridCard">
+                    <section class="GridWrapper">
+                        <section class="second-section">
+                            <div class="images">
+                                <img src="./images/Snapshoot-Portfolio-(3).jpg" alt="section-eader" class="img1"><img
+                                    src="./images/Snapshoot-Portfolio.jpg" alt="img" class="img2">
+                            </div>
+                            <div class="sub-section">
+                                <h2>Tonic</h2>
+
+                                <div class="to-do">
+                                    <ul>
+                                        <li class="canopy">CANOPY</li>
+                                        <li><img src="./images/Counter.jpg" alt="seperation"></li>
+                                        <li class="bck">Back End Dev</li>
+                                        <li><img src="./images/Counter.jpg" alt="seperation"></li>
+                                        <li class="bck">2015</li>
+                                    </ul>
+                                </div>
+                                <p class="to-do-para">A daily selection of privately personalised reads: no accounts or
+                                    sign-ups
+                                    required.</p>
+
+                                <div class="works">
+                                    <ul>
+                                        <li class="list"><a href="#">html</a></li>
+                                        <li class="list"><a href="#">css</a></li>
```

Comment:
- [ ] OPTIONAL: I recommend you setting the font weight of languages to 500 as provided in figma to make your portfolio design look even better: 

Your design  | Figma design
------------- | -------------
<img width="179" alt="Screenshot 2022-02-25 at 15 01 46" src="https://user-images.githubusercontent.com/77038610/155704240-a1840cd4-1784-4305-89bf-256efc211e1e.png">  | <img width="154" alt="Screenshot 2022-02-25 at 15 02 06" src="https://user-images.githubusercontent.com/77038610/155704277-baa74cbe-5fb8-4d80-b991-e59a5a9397dc.png">